viewer/arvviewer.c

```diff
@@ -1117,8 +1117,9 @@ activate (GApplication *application)
 
 	ui_filename = g_build_filename (ARAVIS_DATA_DIR, "arv-viewer.ui", NULL);
 
-	if (!gtk_builder_add_from_file (builder, ui_filename, NULL))
-		g_error ("The user interface file is missing ('%s')", ui_filename);
+	GError *err = NULL;
```

Comment:
Variable declaration must happen at the beginning of the function.

err must be freed using g_error_free.